src/gui/MainWindow.cpp

```diff
@@ -464,6 +464,11 @@ void MainWindow::setMenuActionState(DatabaseWidget::Mode mode)
             m_ui->actionDatabaseMerge->setEnabled(m_ui->tabWidget->currentIndex() != -1);
 
             m_searchWidgetAction->setEnabled(true);
+
+            //if not already in search mode focus the search widget and select all its content
+            if (!inSearch) {
+                m_ui->toolBar->findChild<SearchWidget *>("SearchWidget")->searchFocus();
```

Comment:
Ok, thank you! I'm learning Qt and still digging in code. Sorry I've wasted your time.